homeassistant/helpers/intent.py

```diff
@@ -7,14 +7,17 @@
 from homeassistant.core import callback
 from homeassistant.exceptions import HomeAssistantError
 from homeassistant.loader import bind_hass
+from homeassistant.const import ATTR_ENTITY_ID
 
+REQUIREMENTS = ['fuzzywuzzy==0.16.0', 'python-Levenshtein==0.12.0']
```

Comment:
Helpers can't have requirements.

Reply:
Yeah, i saw that, but the problem in this particular case is that component/__init__.py isn't considered a module so doesn't allow requirements. I could put it anywhere under components but that raises a dependency issue.